Changelog.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this Julia package will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.3.9] – 4. December 2023
```

Comment:
I'm not sure adding third date format for changelog in a single org is a good idea :sweat_smile: